gtfs-validator/main/src/main/java/org/mobilitydata/gtfsvalidator/notice/DecreasingStopTimeDistanceNotice.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.mobilitydata.gtfsvalidator.notice;
+
+import com.google.common.collect.ImmutableMap;
+
+public class DecreasingStopTimeDistanceNotice extends Notice {
+  public DecreasingStopTimeDistanceNotice(
+      String tripId,
+      long csvRowNumber,
+      int stopSequence,
+      double shapeDistTraveled,
+      long prevCsvRowNumber,
+      int prevStopSequence,
+      double prevShapeDistTraveled) {
+    super(
+        new ImmutableMap.Builder<String, Object>()
```

Comment:
You can directly use "ImmutableMap.of(...)" here, to be consistent with all other notices.
No need to have ".put(...)" and ".build()" then.

Reply:
No, not all notices are like this.  "ImmutableMap.of(...)" is limited to 5 entires. See DecreasingShapeDistanceNotice.java that uses the .put and .build

Reply:
Sorry that I did not realised that earlier.